setup.py

```diff
@@ -44,4 +44,5 @@
         'shapely',
         'scipy',
     ],
+    extras_require={'dev': ['ipython', 'tox']},
```

Comment:
What are you using ipython for?

Reply:
It's nice to have for the `./manage.py shell` interface when debugging

Reply:
Exactly. Particularly for `./manage.py shell_plus`.

I'd like to add some docs on these development tools and techniques at some point.